src/main/java/org/folio/circulation/domain/LoanRepository.java

```diff
@@ -335,4 +335,17 @@ private Request matchLoansToRequest(
       .withLoan(loanMap.getOrDefault(request.getItemId(), null));
   }
 
+  public CompletableFuture<Result<MultipleRecords<Loan>>> findOpenLoansByUserId(
```

Comment:
I think this name is misleading, it suggests that it gets all of the open loans for a user.

```suggestion
  public CompletableFuture<Result<MultipleRecords<Loan>>> findOpenLoansByUserIdAndLoanPolicyId(
```

Reply:
renamed